paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -553,18 +551,11 @@ internal class PaymentSheetViewModel @Inject internal constructor(
                     )
                 }
 
-                runCatching {
-                    stripeIntentValidator.requireValid(stripeIntent)
```

Comment:
I don’t see a reason for us to validate the intent after a payment/setup attempt.

Reply:
It’s the same intent that we loaded initially, so it already passed a validation.